how-i-can/news
# PER-5: Click Home Button on Bottom Navigation Bar

**Test Plan**

- Open Perch
- Note down the default set of articles displayed on the home page
- Search for an article or keywords in the search bar
- Click on an article title suggestion
- Click the home button on the bottom navigation bar
- Confirm that the articles shown are reset to the default article set from when you launched Perch
